Automattic/vip-go-mu-plugins
# Two Factor: re-enable FIDO UTF factor

The upstream bug which allowed bypass has been fixed.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Verify that FIDO UTF can now be selected as an option for 2fa
1. If possible, go through the flow to wire up a key and make sure it works (including logging in).
